pkg/operator/ceph/csi/spec.go

```diff
@@ -291,16 +288,18 @@ func startDrivers(clientset kubernetes.Interface, namespace string, ver *version
 		tp.RBDPluginUpdateStrategy = rollingUpdate
 	}
 
-	if ver.Major > KubeMinMajor || (ver.Major == KubeMinMajor && ver.Minor < provDeploymentSuppVersion) {
+	logger.Infof("Kubernetes version is %q.%q", ver.Major, ver.Minor)
```

Comment:
```suggestion
	logger.Infof("Kubernetes version is %s.%s", ver.Major, ver.Minor)
```

Otherwise we will get "14"."1", which seems strange.

Reply:
done :facepalm: not sure why i added  `%q`